microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,528 @@
+---
+title: Get started with WebView2 in HoloLens 2 Unity apps
+description: Get started with WebView2 in HoloLens 2 Unity apps.
+author: MixedRealityTeam
+ms.author: tdb
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 04/17/2023
+---
+# Get started with WebView2 in HoloLens 2 Unity apps (Preview)
+
+> [!NOTE]
+> WebView2 on HoloLens 2 and the WebView Unity plugin are both in Preview and are subject to change before general availability.
+
+In this tutorial, you:
+
+* Set up your development tools for creating HoloLens 2 Unity apps that use WebView2 to display web content.
+* Install the Mixed Reality Toolkit (MRTK) using the MRTK Feature Tool.
+* Create an initial Unity project for HoloLens 2 development.
+* Add the Microsoft Mixed Reality WebView plugin.
+* Configure a WebView2 prefab that displays webpage content in your HoloLens 2 app.
+* Learn about WebView2 concepts and interactions in Unity.
+
+> [!NOTE]
+> *This tutorial is for developers using WebView2 in immersive HoloLens 2 Unity applications. For WebView2-enabled 2D applications on HoloLens 2, refer to the [WinUI 2 (UWP) tutorial](winui2).*
+
+You use the Unity game engine and Visual Studio to create your HoloLens 2 project. The Mixed Reality Toolkit (MRTK) provides essential components and features for developing mixed reality experiences, while the Microsoft Mixed Reality WebView plugin enables the integration of WebView2 functionality into your HoloLens 2 app.
+
+The MRTK is a Microsoft-driven open source project that accelerates the development of mixed reality applications in Unity. This toolkit provides a collection of scripts, components, and prefabs designed to help you create performant and intuitive mixed reality experiences for HoloLens 2 devices.
+
+> [!IMPORTANT]
+> WebView2 is only supported on HoloLens 2 devices running the Windows 11 update. For more information, see [Update HoloLens 2](../../../hololens/hololens-update-hololens).
```

Comment:
```suggestion
> WebView2 is only supported on HoloLens 2 devices running the Windows 11 update. For more information, see [Update HoloLens 2](../../../hololens/hololens-update-hololens.md).
```
not finding that .md yet but will need extension .md if in same repo.  if URL is live now, paste in this comment thread

Reply:
now reads:
```
WebView2 is only supported on HoloLens 2 devices running the Windows 11 update. For more information, see [Update HoloLens 2](/hololens/hololens-update-hololens).
```
you can close this thread